packages/trader/src/AppV2/Components/TradeParamErrorSnackbar/trade-param-error-snackbar.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { SnackbarController, useSnackbar } from '@deriv-com/quill-ui';
+import { getDisplayedContractTypes } from 'AppV2/Utils/trade-types-utils';
+import { useTraderStore } from 'Stores/useTraderStores';
+import useTradeParamError, { TTradeParams } from '../../Hooks/useTradeParamError';
+
+const TradeParamErrorSnackbar = observer(
```

Comment:
Current problems:
1. Each trade param has it's own logic for triggering a snackbar (code duplications)
2. Because at the same moment we have 2 instances of the same component at the page (e.g. stake big and stake minimized), BOTH of them triggers snackbar. As a result there are a lot of bugs, where same error is shown twice.

What I suggest: 
1. To create a component for trade param error snackbar (`TradeParamErrorSnackbar`) and keep all the logic of triggering/showing the snackbar there. Not in multiple places across all the trade params.
2. To render this component in top level, in _trade.tsx_ for e.g. In this case, even if we have 2 instances of the same component with the same error, we will have only one snackbar.
3. To create a hook (`useTradeParamError`), which will incapsulate the logic of errors checking. We will just pass fields, which it should track (e.g. `['take_profit', 'stop_loss']`) and get from hook info if this field has error and what is the error text.
Currently, I'll try to introduce this change for take profit and stop loss only, but if it works well, we can reuse it for duration, barrier and stake.